File: app/src/main/java/com/quran/labs/androidquran/ui/preference/QuranHeaderPreference.java
package com.quran.labs.androidquran.ui.preference;

import com.quran.labs.androidquran.R;

import android.annotation.TargetApi;
import android.content.Context;
import android.graphics.Color;
import android.os.Build;
import androidx.preference.Preference;
import androidx.preference.PreferenceViewHolder;

import android.util.AttributeSet;
import android.widget.TextView;

public class QuranHeaderPreference extends Preference {

  @TargetApi(Build.VERSION_CODES.LOLLIPOP)
  public QuranHeaderPreference(Context context, AttributeSet attrs,
      int defStyleAttr, int defStyleRes) {
    super(context, attrs, defStyleAttr, defStyleRes);
    init();
  }

  public QuranHeaderPreference(Context context, AttributeSet attrs, int defStyleAttr) {
    super(context, attrs, defStyleAttr);
    init();
  }

  public QuranHeaderPreference(Context context, AttributeSet attrs) {
    super(context, attrs);
    init();
  }

  public QuranHeaderPreference(Context context) {
    super(context);
    init();
  }

  @Override
  public void onBindViewHolder(PreferenceViewHolder holder) {
    super.onBindViewHolder(holder);
    if (isEnabled()) {
      final TextView tv = (TextView) holder.findViewById(android.R.id.title);
      if (tv != null) {
        tv.setTextColor(Color.WHITE);
      }
    }
  }

  private void init() {
    setLayoutResource(R.layout.about_header);
    setSelectable(false);
  }
}

File: app/src/main/java/com/quran/labs/androidquran/ui/preference/QuranHeaderPreference.kt
package com.quran.labs.androidquran.ui.preference

import android.content.Context
import android.graphics.Color
import android.util.AttributeSet
import android.widget.TextView
import androidx.preference.Preference
import androidx.preference.PreferenceViewHolder
import com.quran.labs.androidquran.R

class QuranHeaderPreference @JvmOverloads constructor(
  context: Context,
  attrs: AttributeSet? = null,
  defStyleAttr: Int = 0,
  defStyleRes: Int = 0,
) : Preference(context, attrs, defStyleAttr, defStyleRes) {

  init {
    layoutResource = R.layout.about_header
    isSelectable = false
  }

  override fun onBindViewHolder(holder: PreferenceViewHolder) {
    super.onBindViewHolder(holder)
    if (isEnabled) {
      val tv = holder.findViewById(R.id.title) as? TextView
      tv?.setTextColor(Color.WHITE)
    }
  }
}
